cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -120,6 +130,9 @@ public Cube get() {
     aggs.put("flow.queue", new DefaultAggregation(
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.FLOW, Constants.Metrics.Tag.CONSUMER,
+                       Constants.Metrics.Tag.PRODUCER, Constants.Metrics.Tag.FLOWLET_QUEUE),
+      ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
+                       Constants.Metrics.Tag.FLOW, Constants.Metrics.Tag.CONSUMER,
```

Comment:
well, while testing and debugging - found that we do write too many data than needed - i.e. that filter was absent. It needs to be there. Fine within this PR:)